ods_ci/tests/Resources/Files/triton/triton_pytorch_rest_servingruntime.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  name: kserve-triton-pytorch-rest
+spec:
+  annotations:
+    prometheus.kserve.io/path: /metrics
+    prometheus.kserve.io/port: "8002"
```

Comment:
you guys are able to et the metric endpoint i am asking this because the serving port is different